standard/expressions.md

```diff
@@ -4828,11 +4882,13 @@ The simplest form of an anonymous function is one that captures no outer variabl
 
 <!-- UsingsExample: {template:"standalone-lib", name:"AnonFunctionImplementationExample1"} -->
 ```csharp
+delegate void D();
+
 class Test
 {
     static void F()
     {
-        D d = () => Console.WriteLine("test");
+        D d = () => System.Console.WriteLine("test");
```

Comment:
Again, revert this?

Reply:
(And a few changes below, at lines 4911, 4928, 4950, 4992, 5014, 5055.)

Reply:
All reversed